docs/assets/css/style.css

```diff
@@ -8,6 +8,14 @@ html {
 *, *::after, *::before {
   box-sizing: inherit; }
 
+.discord-logo:before {
```

Comment:
This one is my bad. Should be 
`.discord-logo::before {`

Reply:
Are you sure? Other instances in this file use `:before`.

Reply:
Hmm, I double checked. Seems like both are fine. See second answer of [this](https://stackoverflow.com/questions/7327326/what-is-the-difference-between-before-and-before)